### DISCUSSION TITLE
https://johanneskonings.dev/blog/2024-04-20-cdk-notifier-and-tags/

### DISCUSSION THREAD

**Discussion by giscus** [JohannesKonings/JohannesKonings.github.io#24] [Blog Post Comments]
# https://johanneskonings.dev/blog/2024-04-20-cdk-notifier-and-tags/

Description of the diff consequence of two ways of tagging in CDK

https://johanneskonings.dev/blog/2024-04-20-cdk-notifier-and-tags/

**Comment by JohannesKonings** [comment]
test
